### DISCUSSION TITLE
Login fails after migrating

### DISCUSSION THREAD

**Discussion by 43615** [netbox-community/netbox#14262] [Help Wanted!] [answered]
Migrated from a testing system to a proper server using the documented dump/import commands. Server runs, but I can't log in due to an error that looks completely unrelated:
```
<class 'netaddr.core.AddrFormatError'>

failed to detect a valid IP address from ''

Python version: 3.10.12
NetBox version: 3.6.5
Plugins: None installed
```
Looks like some incorrect data in the DB (which is also bad), but I don't see how that's related to logging in.

**Comment by 43615** [comment, marked as answer]
Turns out that something was wrong with the superuser, maybe because I reused the name from the old instance? Created a new one, everything's fine.
Still unsure how that error is related, but marking as closed.

**Comment by r0bc94** [comment]
I'm still facing this issue when trying to remove the remote authentication. After removing the relevant parts out of the `configuration.py` file, this error is shown when trying to log in with the local user account.
